packages/react-resizable-panels/src/env-conditions/unknown.ts

```diff
@@ -0,0 +1 @@
+export const isBrowser = typeof window !== "undefined";
```

Comment:
This kinda should almost never be even chosen but better to be safe than sorry, right? :p